src/console/Normalize/ProcessorStartup.cs

```diff
@@ -0,0 +1,36 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using EnsureThat;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Health.Fhir.Ingest.Config;
+
+namespace Microsoft.Health.Fhir.Ingest.Console.Normalize
+{
+    public class ProcessorStartup
+    {
+        public ProcessorStartup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        public void ConfigureServices(IServiceCollection services)
+        {
+            var outputEventHubConnection = Configuration.GetSection("OutputEventHub").Value;
+            var outputEventHubName = outputEventHubConnection.Substring(outputEventHubConnection.LastIndexOf('=') + 1);
```

Comment:
>var outputEventHubName = outputEventHubConnection.Substring(outputEventHubConnection.LastIndexOf('=') + 1); [](start = 12, length = 107)

Should be able to use the event hub connection string builder here.

Reply:
Yes, once this is in I am planning on making stories for supporting MI.  There are few things I would like to iterate on overall in this code but I would like to this initial build in as a base committed to master so we have the history.

---
In reply to: [544605846](https://github.com/microsoft/iomt-fhir/pull/78#discussion_r544605846) [](ancestors = 544605846)